libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/ModelOverview/ChartConfigurationFlyout.tsx

```diff
@@ -68,9 +68,9 @@ export class ChartConfigurationFlyout extends React.Component<
       // maximum which indicates that new cohorts were created.
       newCohorts = this.props.datasetCohorts
         .filter(
-          (errorCohort) => errorCohort.cohort.getCohortID() > prevMaxCohortID
+          (errorCohort) => errorCohort?.cohort.getCohortID() > prevMaxCohortID

```

Comment:
will `cohort` be guaranteed to be defined? Do you need to do `errorCohor?.cohort?.getCohortID()` ?